components/src/core/selection-toolbar/selection-toolbar.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, ViewEncapsulation, Input, ViewChild, ElementRef } from '@angular/core';
+import { isEmptyView } from '../../utils/utils';
+import { MatMenuTrigger } from '@angular/material/menu';
+
+@Component({
+    selector: 'pxb-selection-toolbar',
+    encapsulation: ViewEncapsulation.None,
+    template: `
+        <mat-toolbar fxLayout="row" class="pxb-selection-toolbar">
+            <div
+                #icon
+                class="pxb-selection-toolbar-icon-wrapper"
+                [class.pxb-selection-toolbar-no-icon]="isIconEmpty(iconEl)"
+            >
+                <ng-content select="[pxb-icon]"></ng-content>
+            </div>
+            <div *ngIf="title" class="pxb-selection-toolbar-title-wrapper">
+                <div class="pxb-selection-toolbar-title">{{ title }}</div>
+                <div
+                    *ngIf="subtitle"
+                    class="pxb-selection-toolbar-subtitle-wrapper mat-subheading-2"
+                    [matMenuTriggerFor]="selectionToolbarMenu"
+                    #menuTrigger="matMenuTrigger"
+                    style="cursor: pointer;"
+                >
+                    <span class="pxb-selection-toolbar-subtitle">{{ subtitle }}</span>
```

Comment:
Is there a mat-typography element you can use here instead?

Reply:
There isn't a mat-typography element that works out nicely for this.

Reply:
How far off are they?

Reply:
Not too far off for title, it's mostly line-height that is an issue. I didn't even try one for subtitle because none of the configurations look close. I'm open to recommendations on what to use here if we want to use mat-typography here.